doc/content/resources/neams-workbench.md

```diff
@@ -0,0 +1,162 @@
+remote: Counting objects: 100% (885/885), done.
+remote: Compressing objects: 100% (366/366), done.
+remote: Total 4524 (delta 534), reused 719 (delta 502), pack-reused 3639
+Receiving objects: 100% (4524/4524), 112.31 MiB | 39.01 MiB/s, done.
+Resolving deltas: 100% (2487/2487), done.
+Downloading htgr/htr10/data/mesh/htr-10-critical-a-rev6.e (2.7 MB)
+Downloading htgr/htr10/data/mesh/htr-10-full-a-rev3.e (3.5 MB)
+Downloading htgr/htr10/data/sph/htr-10-Diff-SPH.xml (3.0 MB)
+Downloading htgr/htr10/data/xs/htr-10-XS.xml (4.9 MB)
+Downloading htgr/htr10/steady/gold/out-HTR-10-full-Diff-SPH-ARO.e (7.4 MB)
+Downloading htgr/htr10/steady/gold/out_HTR-10_critical.e (7.9 MB)
+Downloading htgr/pbmr400/shared/oecd_pbmr400_tabulated_xs.xml (212 MB)
+Downloading mrad/mesh/mrad_mesh.e (62 MB)
+Downloading pbfhr/reflector/conduction/conduction.fld (247 MB)
+Downloading pbfhr/reflector/meshes/fluid.re2 (60 MB)
+Downloading pbfhr/reflector/meshes/solid.e (62 MB)
+Checking out files: 100% (370/370), done.
+```
+
+The content of the `virtual_test_bed` directory can be accessed with the `ls` command:
+
+```console
+ls virtual_test_bed/
+apps  COPYRIGHT  doc  htgr  LICENSE  mrad  msr  pbfhr  README.md  scripts  sfr
+```
+
+All VTB examples are run with the *BlueCrab* application (see Section 5 for further details); users are referred to Section 4 for details.
+
+# Using the NEAMS Workbench to run a BlueCrab multiphysics model on Sawtooth
+
+## Conventional Workflow on an HPC Platform
+
+Conventionally, a modeling and simulation application is run on an HPC platform in a terminal by following four steps.
+
+- Step 1: connect to the remote HPC platform using [Secure Shell](https://www.ssh.com/academy/ssh) or one of its derivatives.
+- Step 2: enter the working directory of choice, then edit the input file and any other relevant files, including the mesh files.
+- Step 3: submit the job to the scheduler using a submission script from the working directory. The submission script contains logic to load the correct     environment as well as the executable to run and the number of cores to use.
+- Step 4: once the job is completed, visualize and analyze the numerical solution using a data visualization application, such as VisIt or ParaView. Data visualization may require copying large result files from the cluster to a desktop environment.
+
+This workflow can be cumbersome to users unfamiliar with HPC platforms and an obstacle to potential industrial users in the nuclear engineering field who  want to leverage HPC resources to speed up calculations. NCRC intends to increase the accessibility of modeling and simulation codes and the use of HPC resources by allowing users to access an HPC login node through a virtual desktop from the login page of the HPC OnDemand website (step 1). From there, users  can run an application (*BlueCrab*) or open a terminal to edit files but still need their own submission script to submit jobs to the scheduler.
```

Comment:
This is great for explaining why Workbench is useful, I dont think the typical VTB user needs to see the justification NEAMS has for using Workbench

Reply:
My understanding is that documentation is to be used by industry users at some point in the near future and they may not be as proficient as the current VTB users/developers.

Reply:
but do they need to know about the conventional workflow here, when we're trying to show the workbench workflow